features/core-language.feature

```diff
@@ -16,6 +17,8 @@ Feature: Manage translation files for a WordPress install
     And I run `wp core language install en_AU`
     Then the wp-content/languages/admin-en_GB.po file should exist
     And the wp-content/languages/en_GB.po file should exist
+    And the {SUITE_CACHE_DIR}/translation/core-default-([\d\.]+)-en_AU-([\d]+).zip file should exist
+    And the {SUITE_CACHE_DIR}/translation/core-default-([\d\.]+)-en_GB-([\d]+).zip file should exist
```

Comment:
@ocean90 the `file should exist` step doesn't accept regex for the filename, afaik.

You could instead run `grep` on the translation cache dir to verify the files are there. Pseudo code:

```
When I run `ls {SUITE_CACHE_DIR}/translation | grep core-default-`
Then STDOUT should contain:
  """
  en_AU
  """
And STDOUT should contain:
  """
  en_GB
  """
```